content/post/build-code-ios.md

```diff
@@ -57,6 +57,16 @@ PYTHON MODULES and NODEJS
 
 2.1.2) Install nodejs from https://nodejs.org/en/download (macOS pkg). This package provides `npm` and `node` commands that are required to build everything in `browser/` folder.
 
+2.1.3) Install Homebrew from https://github.com/Homebrew/brew/releases/latest (macOS pkg) and add /opt/homebrew/bin and /opt/homebrew/sbin to the end of your PATH.
```

Comment:
Might be nice to provide commands for people to do this ... maybe not necessary given the target audience of this post though. Definitely not going to block the change based on that (if we do it at all we can do it in a followup)